src/blockchain/log-processors/order-created.ts

```diff
@@ -1,19 +1,33 @@
-import { Database } from "sqlite3";
+import * as Knex from "knex";
 import { FormattedLog, ErrorCallback } from "../../types";
 
-export function processOrderCreatedLog(db: Database, log: FormattedLog, callback: ErrorCallback): void {
-  db.get(`SELECT block_timestamp FROM blocks WHERE block_number = ?`, [log.blockNumber], (err?: Error|null, blocksRow?: {block_timestamp: number}): void => {
+export function processOrderCreatedLog(db: Knex, log: FormattedLog, callback: ErrorCallback): void {
+  db.raw("select block_timestamp from blocks where block_number = ?", [log.blockNumber])
```

Comment:
Recommend using consistent casing for SQL (SELECT, FROM, etc. are capitalized in other statements).

Reply:
Done.